src/test/java/core/basesyntax/dao/impl/StorageDaoImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    private final Fruit banana = new Fruit("banana");
+    private final Fruit apple = new Fruit("apple");
+    private final int quantity = 50;
+
+    @AfterEach
+    void setUp() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    void addProductToEmptyStorage_Ok() {
+        StorageDao storageDao = new StorageDaoImpl();
+        storageDao.add(banana, quantity);
+        Integer expected = 50;

```

Comment:
maybe you should use constants for all the numbers, so code could look cleaner

Reply:
you don't need to use constants in tests